cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js

```diff
@@ -221,10 +221,6 @@ describe('ui-users-loans: renewal failure because loan has reached maximum renew
       InventoryInstance.deleteInstanceViaApi(secondInstanceIds.instanceId);
     });
 
-    cy.updateCirculationRules({
```

Comment:
'rulesDefaultString' is assigned a value but never used. Allowed unused vars must match /React/u

Reply:
Deleted